clang/lib/Gold/GoldDependentExprTransformer.cpp

```diff
@@ -118,12 +118,15 @@ clang::Expr *DependentExprTransformer::transformCppxDependentMemberAccessExpr(
 
   clang::Expr *Ret = transformDependentExpr(E->getBase());
   if (!Ret) {
+    llvm::outs() << "We have an invalid dependent expression that was evaluated!\n";
```

Comment:
use a custom diagnostic, they're all over the ExprElaborator. and please use language that actually sounds like a diagnostic.

Reply:
Fixed but not pushed.